mgmtfn/dockplugin/dockplugin.go

```diff
@@ -307,10 +319,15 @@ func createEndpoint(tenantName, hostname string) func(http.ResponseWriter, *http
 			return
 		}
 
-		// FIXME:
-		GetEndPointName(cereq.NetworkID, cereq.EndpointID)
+		// Get Service name
+		serviceName, err := GetEndPointName(cereq.NetworkID, cereq.EndpointID)
```

Comment:
Yeah, more or less. Sorry for the nit!

-Erik

On 17 Sep 2015, at 20:01, Sukhesh Halemane wrote:

>> @@ -307,10 +319,15 @@ func createEndpoint(tenantName, hostname 
>> string) func(http.ResponseWriter, *http
>> 			return
>> 		}
>>
>> -		// FIXME:
>> -		GetEndPointName(cereq.NetworkID, cereq.EndpointID)
>> +		// Get Service name
>> +		serviceName, err := GetEndPointName(cereq.NetworkID, 
>> cereq.EndpointID)
>
> Are you saying 'P' should not be capitalized in Endpoint?
> I think it was capitalized by accident and then got cut and pasted..
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/netplugin/pull/148/files#r39820746